static/ankersrv.js

```diff
@@ -119,35 +160,30 @@ $(function () {
 
         _open() {
             $(this.badge).removeClass("text-bg-success text-bg-danger").addClass("text-bg-warning");
-            if (this.open)
-                this.open(this.ws);
+            if (this.open) this.open(this.ws);
         }
 
         _close() {
             $(this.badge).removeClass("text-bg-warning text-bg-success").addClass("text-bg-danger");
             console.log(`${this.name} close`);
             setTimeout(() => this.connect(), this.reconnect);
-            if (this.close)
```

Comment:
Can we revert these autoformatting changes as well? Thanks :)

Reply:
✅